Makefile

```diff
@@ -167,6 +168,15 @@ unit-test: unit-tests
 int-tests: gotools fabric-ca-server fabric-ca-client
 	@scripts/run_integration_tests
 
+vendor: .FORCE
```

Comment:
@sykesm  - I'm sure there's a cleaner way to do this, but this seems to work and leaves us without a `go.mod` and `go.sum`  in the root of the repo